ext/mixed/js/text-scanner.js

```diff
@@ -296,10 +296,8 @@ class TextScanner extends EventDispatcher {
                 return;
             }
 
-            for (const ignorePointFn of this._ignorePoints) {
-                if (await ignorePointFn(x, y)) {
-                    return;
-                }
+            if (typeof this._ignorePoints === 'function' && await this._ignorePoints(x, y)) {
```

Comment:
This could be called `ignorePoint` now that there's only one.

Reply:
45d6cf49b3423dfdc8f6d7c19553b5a65c38bb58

Reply:
Looks good :+1: